website/content/docs/ecs/deploy/manual.mdx

```diff
@@ -0,0 +1,337 @@
+
+Provide Consul server connection settings to the mesh task module so that the module can configure the control-plane and ECS controller containers to connect to the servers. 
+
+1. In your `variables.tf` file, define variables for the host URL and TLS settings for gRPC and HTTP traffic. Refer to the [mesh task module reference](https://registry.terraform.io/modules/hashicorp/consul-ecs/aws/latest/submodules/gateway-task?tab=inputs) for information about the variables you can define. In the following example, the Consul server address is defined in the `consul_server_hosts` variable:
+
+  ```hcl
+  variable "consul_server_hosts" {
+    description = "Address of Consul servers."
+    type        = string
+  }
+
+  ```
+1. Add an `environment` block to the control-plane and ECS controller containers definition 
+1. Set the `environment.name` field to the `CONSUL_ECS_CONFIG_JSON` environment variable and the value to `local.encoded_config`.
+
+  ```hcl
+  environment = [
+    {
+      name  = "CONSUL_ECS_CONFIG_JSON",
+      value = local.encoded_config
+    }
+  ]
+  ```
+  When you apply the configuration, the mesh task module interpolates the server configuration variables, builds a `config.tf` file, and injects the settings into the appropriate containers.
+
+For additional information about the `config.tf` file, refer to the [JSON schema reference documentation](/consul/docs/ecs/reference/config-json-schema). 
+
+## Register the task definition configuration
+
+Register the task definition with your ECS cluster using the AWS Console, AWS CLI, or another method supported by AWS. You must also create an ECS Service to start tasks using the task definition. Refer to the following ECS documentation for information on how to register the task definition:
+
+- [Creating a task definition using the classic console](https://docs.aws.amazon.com/AmazonECS/latest/userguide/create-task-definition-classic.html)
+- [Register task definition CLI](https://docs.aws.amazon.com/cli/latest/reference/ecs/register-task-definition.html)
+
+## Deploy the controller container
+
+The controller container runs in a separate ECS task and is responsible for Consul security features. The controller uses the AWS IAM auth method to enable ECS tasks to automatically obtain Consul ACL tokens when the task starts up. Refer to [Consul security components](/consul/docs/ecs/architecture#consul-security-components) for additional information.
+
+Verify that you have completed the prerequisites described in [Secure configuration requirements](#secure-configuration-requirements) and configure the following components to enable Consul security features.
+
+### Create an ACL policy
+
+On the Consul server, create a policy that grants `acl:write` and `operator:write` access for the controller. The policy allows Consul to generate a token linked to the policy. Refer to [Create a service token](/consul/docs/security/acl/tokens/create/create-a-service-token) for instructions.
```

Comment:
This should also include the `node:write` policy because the controller container registers a synthetic node.